test/fixtures/dialects/snowflake/snowflake_select_where_is_distinct_from.sql

```diff
@@ -0,0 +1,9 @@
+SELECT
+    a,
+    b
+FROM person where a IS DISTINCT FROM b;
+
+SELECT
+    a,
+    b
+FROM person where a IS NOT DISTINCT FROM b;
```

Comment:
Lets have a trailing newline:

```suggestion
FROM person where a IS NOT DISTINCT FROM b;

```